common/tests/notification_template_fixtures.py

```diff
@@ -13,6 +13,10 @@
     Occurrence: {{ occurrence.local_start_time.strftime('%d.%m.%Y %H.%M') }}
     {% endif%}
     Person: {{ study_group.person.email_address}}
+    {% if occurrence.p_event.needed_occurrences == 1 %}
+    Click this link to cancel the enrolment:
+    {{ enrolment.get_link_to_cancel_ui()}}
```

Comment:
In the pull request #109 this function is named `enrolment.create_cancellation_url()`. That returns a string that starts with configuration variable `settings.VERIFICATION_TOKEN_URL_MAPPING.get("appname.model").get("CANCELLATION")` and is appended with `cancellation_token.key`.

So the full configuration is 
```
VERIFICATION_TOKEN_URL_MAPPING = {
    "occurrences.enrolment": {
        "CANCELLATION": f"{KULTUS_TEACHER_UI_BASE_URL}/cancel_enrolment_url/"
    }
}
```